app/views/redirections/_redirection.html.erb

```diff
@@ -0,0 +1,6 @@
+<tr>
+  <%= content_tag(:td, redirection.slug) %>
```

Comment:
I've kind of gotten into the habit of using the `content_tag` and `content_tag_for` methods. I find them somewhat cleaner than switching back and forth between html and erb. I dunno, it's something I'm trying out to see if it's any better. Do you like the other way better or are you just curious?